script/DeployPeriphery.sol

```diff
@@ -4,15 +4,23 @@ pragma solidity ^0.8.13;
 import { BaseDeployPeriphery } from "./BaseDeployPeriphery.sol";
 import { DeployManagers } from "./DeployManagers.sol";
 import { DeployCompanion } from "./DeployCompanion.sol";
-import { DeployStrategies } from "./strategies/aave-v3/base/DeployStrategies.sol";
+import { DeployStrategies as DeployAaveV3Strategies } from "./strategies/aave-v3/base/DeployStrategies.sol";
```

Comment:
It's kind of weird to add specific chain deployments here. I think we should deploy periphery in without strategies here, and then deploy strategies as a separate command. Or we could have a deployPeriphery per chain, as you prefer